bin/migrate-oats-data/menu/commands/applications.py

```diff
@@ -21,9 +25,11 @@ def application_import(console, args):
         init_applications(batch_size=import_batch_size)
         init_alcs_users(batch_size=import_batch_size)
         process_application_etl(batch_size=import_batch_size)
+        populate_default_staff_journal_user()
```

Comment:
staff journal for applications gets populated within process_application_etl, should that be brought out to this level?

Reply:
nope, since the application_staff_journal script creates entries only for applications where this script updates all staff journal records (application, noi, etc.). Also, script designed in a way that it is possible to run it without clearing any data multiple times and nothing will change in output result